addons/mail/static/src/components/attachment_list/attachment_list.xml

```diff
@@ -14,6 +14,11 @@
                         <AttachmentCard className="'o_AttachmentList_attachment'" localId="attachmentCard.localId" onAttachmentRemoved="props.onAttachmentRemoved"/>
                     </t>
                 </div>
+                <div class="o_AttachmentList_partialList o_AttachmentList_preview">
```

Comment:
on line 12, rename classname `o_AttachmentList_partialListNonImages` to `o_AttachmentList_partialListCards`